TA_dataset/bin/dataset_common.py

```diff
@@ -233,13 +217,13 @@ def get_acct_info(self, logger, account=None):
     return acct_dict
 
 
-def get_tenant_value(conf, logger):
-    tenant_value = conf.tenant
-    tenant_value = tenant_value.strip()
-    logger.debug("The provided tenant value in config is {}".format(tenant_value))
-    if tenant_value.lower() == "false" or tenant_value.lower() == "0":
-        return False
-    return True
+def get_updated_account_dict(conf, conf_name, acct_dict, logger):
```

Comment:
Can we have a more descriptive name, e.g. `update_tenant_conf`

Reply:
Done